### ISSUE TITLE
Index out of range 

### ISSUE THREAD

**Issue by ludwich** [pytube/pytube#5]
http://www.youtube.com/watch?v=zLTOzu8N1Pg&feature=youtube_gdata
Traceback (most recent call last):
  File "test2.py", line 35, in <module>
    yt.url = data['feed']['entry'][0]['link'][0]['href']
  File "build/bdist.linux-x86_64/egg/pytube/api.py", line 75, in url
  File "build/bdist.linux-x86_64/egg/pytube/api.py", line 211, in _get_video_info
IndexError: list index out of range

**Comment by hansihe** [comment]
My issue does seem to be the same as this one, so I will provide some more information.
This error only seems to appear occasionally. I would say about 1/10 of attempts.
Here is my full trace:

```
Traceback (most recent call last):
  File "C:\Users\HansiHE\PycharmProjects\YouTubeCards\run.py", line 10, in <module>
    youtube.url = 'http://www.youtube.com/watch?v=kahpg789lGs'
  File "C:\Users\HansiHE\PycharmProjects\YouTubeCards\pytube\api.py", line 74, in url
    self._get_video_info()
  File "C:\Users\HansiHE\PycharmProjects\YouTubeCards\pytube\api.py", line 210, in _get_video_info
    signature = video_signatures[idx]
IndexError: index out of range: 8
```

It occurs on this piece of code:

``` python
from pytube.api import YouTube

youtube = YouTube()
youtube.url = 'http://www.youtube.com/watch?v=kahpg789lGs'
print youtube.filter('mp4', '360p')
```

**Comment by rbrcsk** [comment]
@hansihe I described what's wrong on issue #6. The same error is causing this too.

**Comment by hansihe** [comment]
@randomchars Ah, thanks for letting me know.

**Comment by heetbeet** [comment]
This error seems to persist
everything runs fine with:

```
yt = YouTube()
yt.url = 'http://www.youtube.com/watch?v=kahpg789lGs'
```

but there is a problem with:

```
yt = YouTube()
yt.url = 'https://www.youtube.com/watch?v=PIb6AZdTr-A'
```

```
IndexError                                Traceback (most recent call last)
<ipython-input-8-a8b54bd239d1> in <module>()
----> 1 yt.url = 'https://www.youtube.com/watch?v=PIb6AZdTr-A'

C:\Users\Simon\Dropbox\petprojects\youtube_downloader\pytube\api.pyc in url(self, url)
     74         self._filename = None
     75         #Get the video details.
---> 76         self._get_video_info()
     77
     78     @property

C:\Users\Simon\Dropbox\petprojects\youtube_downloader\pytube\api.pyc in _get_video_info(self)
    239             for idx in range(len(video_urls)):
    240                 url = video_urls[idx]
--> 241                 signature = video_signatures[idx]
    242                 try:
    243                     fmt, data = self._extract_fmt(url)

IndexError: list index out of range
```

it seems that video_signatures is an empty list.

**Comment by nficano** [comment]
fixed is in v0.2.1 going out Feb 8th https://github.com/NFicano/pytube/tree/v0.2.1  
- Nick
